docs/content/latest/explore/ysql-language-features/triggers.md

```diff
@@ -230,10 +229,24 @@ CREATE EVENT TRIGGER tr_name ON event
 
 *tr_name*, which is unique in the database, represents the new trigger. *event* represents the event that triggers a call to the function  *function_name* whose return type is `event_trigger` (optional). You can define more than one trigger for the same event, in which case the triggers fire in alphabetical order based on the name of the trigger. If a `WHEN` condition is included in the `CREATE EVENT TRIGGER` statement, then the trigger is fired for specific commands. *filter_variable* needs to be set to`TAG`, as this is the only supported variable, and *filter_value* represents a list of values for *filter_variable*.
 
-The following example is based on examples from [Creating Triggers](#creating-triggers) and shows how to create an  `sql_drop` trigger for one of the events currently supported by YSQL:
+The following example is based on examples from [Creating Triggers](#creating-triggers), except that the `record_dept_changes` function returns an event trigger instead of a regular trigger. The example shows how to create an  `sql_drop` trigger for one of the events currently supported by YSQL:
+
+```sql
+CREATE OR REPLACE FUNCTION record_dept_changes() 
+RETURNS EVENT_TRIGGER AS
+$$
+BEGIN 
+IF NEW.department <> OLD.department 
+THEN INSERT INTO employee_dept_changes(employee_no, name, department, changed_on) 
+VALUES(OLD.employee_no, OLD.name, OLD.department, now()); 
+END IF; 
+END;
+$$
+LANGUAGE 'plpgsql';
+```
 
 ```sql
 CREATE EVENT TRIGGER dept_changes ON sql_drop
-  EXECUTE PROCEDURE record_dept_changes();
+  EXECUTE PROCEDURE record_dept_changes1();
```

Comment:
Typo?
```suggestion
  EXECUTE PROCEDURE record_dept_changes();
```

Reply:
Yes, thanks for catching this! (I had several of these functions set up and forgot to clean up one spot)